src/neuroconv/datainterfaces/ophys/scanimage/scanimageimaginginterface.py

```diff
@@ -65,9 +72,16 @@ def get_metadata(self) -> dict:
 
         metadata = super().get_metadata()
 
-        if "state.internal.triggerTimeString" in self.image_metadata:
-            extracted_session_start_time = dateparse(self.image_metadata["state.internal.triggerTimeString"])
+        try:
+            extracted_session_start_time = get_metadata_value(
+                self.image_metadata,
+                "state.internal.triggerTimeString",
+                "epoch",
+            )
             metadata["NWBFile"].update(session_start_time=extracted_session_start_time)
+        except ValueError:
+            # If not present, ignore this aspect
+            pass
```

Comment:
Similar to above; we try to avoid using try/except clauses unless absolutely necessary

Would suggest something like

```python
if "state.internal.triggerTimeString" in self.image_metadata:
    extracted_session_start_time = dateparse(self.image_metadata["state.internal.triggerTimeString"])
    metadata["NWBFile"].update(session_start_time=extracted_session_start_time)
elif "epoch" in self.image_metadata:
    extracted_session_start_time = dateparse(self.image_metadata["epoch"])  # or whatever else is necessary to coerce/parse the value
    metadata["NWBFile"].update(session_start_time=extracted_session_start_time)
```

Reply:
Fair enough. I dislike the duplication of the update line, but on the whole, my solution is both more verbose, and less in agreement with the style of the rest of the project

Reply:
The format stored by ScanImage in more recent versions defeats `dateutils.parser.parse`, unfortunately, but `datetime.datetime.strptime` should do the job unless/until the format changes again